### ISSUE TITLE
2025-05-09, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#164]
# Today's Arxiv Papers

(1) [Generating Physically Stable and Buildable LEGO Designs from Text](http://arxiv.org/abs/2505.05469v1), 2025-05-08

(2) [StreamBridge: Turning Your Offline Video Large Language Model into a Proactive Streaming Assistant](http://arxiv.org/abs/2505.05467v1), 2025-05-08

(3) [ComPO: Preference Alignment via Comparison Oracles](http://arxiv.org/abs/2505.05465v1), 2025-05-08

(4) [Bring Reason to Vision: Understanding Perception and Reasoning through Model Merging](http://arxiv.org/abs/2505.05464v1), 2025-05-08

(5) [UKElectionNarratives: A Dataset of Misleading Narratives Surrounding Recent UK General Elections](http://arxiv.org/abs/2505.05459v1), 2025-05-08

(6) [Conversational Process Model Redesign](http://arxiv.org/abs/2505.05453v1), 2025-05-08

(7) [clem:todd: A Framework for the Systematic Benchmarking of LLM-Based Task-Oriented Dialogue System Realisations](http://arxiv.org/abs/2505.05445v1), 2025-05-08

(8) [GesPrompt: Leveraging Co-Speech Gestures to Augment LLM-Based Interaction in Virtual Reality](http://arxiv.org/abs/2505.05441v1), 2025-05-08

(9) [EcoAgent: An Efficient Edge-Cloud Collaborative Multi-Agent Framework for Mobile Automation](http://arxiv.org/abs/2505.05440v1), 2025-05-08

(10) [Ultra-FineWeb: Efficient Data Filtering and Verification for High-Quality LLM Training Data](http://arxiv.org/abs/2505.05427v1), 2025-05-08

(11) [TransProQA: an LLM-based literary Translation evaluation metric with Professional Question Answering](http://arxiv.org/abs/2505.05423v1), 2025-05-08

(12) [Crosslingual Reasoning through Test-Time Scaling](http://arxiv.org/abs/2505.05408v1), 2025-05-08

(13) [Frame In, Frame Out: Do LLMs Generate More Biased News Headlines than Humans?](http://arxiv.org/abs/2505.05406v1), 2025-05-08

(14) [DSDrive: Distilling Large Language Model for Lightweight End-to-End Autonomous Driving with Unified Reasoning and Planning](http://arxiv.org/abs/2505.05360v1), 2025-05-08

(15) [Hearing and Seeing Through CLIP: A Framework for Self-Supervised Sound Source Localization](http://arxiv.org/abs/2505.05343v1), 2025-05-08

(16) [FLAM: Frame-Wise Language-Audio Modeling](http://arxiv.org/abs/2505.05335v1), 2025-05-08

(17) [ICon: In-Context Contribution for Automatic Data Selection](http://arxiv.org/abs/2505.05327v1), 2025-05-08

(18) [Toward Reasonable Parrots: Why Large Language Models Should Argue with Us by Design](http://arxiv.org/abs/2505.05298v1), 2025-05-08

(19) [PlaceIt3D: Language-Guided Object Placement in Real 3D Scenes](http://arxiv.org/abs/2505.05288v1), 2025-05-08

(20) [HEXGEN-TEXT2SQL: Optimizing LLM Inference Request Scheduling for Agentic Text-to-SQL Workflow](http://arxiv.org/abs/2505.05286v1), 2025-05-08

(21) [Software Development Life Cycle Perspective: A Survey of Benchmarks for CodeLLMs and Agents](http://arxiv.org/abs/2505.05283v1), 2025-05-08

(22) [PADriver: Towards Personalized Autonomous Driving](http://arxiv.org/abs/2505.05240v1), 2025-05-08

(23) [Latte: Transfering LLMs` Latent-level Knowledge for Few-shot Tabular Learning](http://arxiv.org/abs/2505.05237v1), 2025-05-08

(24) [ChemRxivQuest: A Curated Chemistry Question-Answer Database Extracted from ChemRxiv Preprints](http://arxiv.org/abs/2505.05232v1), 2025-05-08

(25) [QualBench: Benchmarking Chinese LLMs with Localized Professional Qualifications for Vertical Domain Evaluation](http://arxiv.org/abs/2505.05225v1), 2025-05-08

(26) [Stealthy LLM-Driven Data Poisoning Attacks Against Embedding-Based Retrieval-Augmented Recommender Systems](http://arxiv.org/abs/2505.05196v1), 2025-05-08

(27) [Revealing Weaknesses in Text Watermarking Through Self-Information Rewrite Attacks](http://arxiv.org/abs/2505.05190v1), 2025-05-08

(28) [Biomed-DPT: Dual Modality Prompt Tuning for Biomedical Vision-Language Models](http://arxiv.org/abs/2505.05189v1), 2025-05-08

(29) [MARK: Memory Augmented Refinement of Knowledge](http://arxiv.org/abs/2505.05177v1), 2025-05-08

(30) [FedTDP: A Privacy-Preserving and Unified Framework for Trajectory Data Preparation via Federated Learning](http://arxiv.org/abs/2505.05155v1), 2025-05-08

(31) [Text2Cypher: Data Pruning using Hard Example Selection](http://arxiv.org/abs/2505.05122v1), 2025-05-08

(32) [Enhancing Text2Cypher with Schema Filtering](http://arxiv.org/abs/2505.05118v1), 2025-05-08

(33) [Unveiling Language-Specific Features in Large Language Models via Sparse Autoencoders](http://arxiv.org/abs/2505.05111v1), 2025-05-08

(34) [Multi-agent Embodied AI: Advances and Future Directions](http://arxiv.org/abs/2505.05108v1), 2025-05-08

(35) [A Weighted Byzantine Fault Tolerance Consensus Driven Trusted Multiple Large Language Models Network](http://arxiv.org/abs/2505.05103v1), 2025-05-08

(36) [X-Driver: Explainable Autonomous Driving with Vision-Language Models](http://arxiv.org/abs/2505.05098v1), 2025-05-08

(37) [Reliably Bounding False Positives: A Zero-Shot Machine-Generated Text Detection Framework via Multiscaled Conformal Prediction](http://arxiv.org/abs/2505.05084v1), 2025-05-08

(38) [Performance Evaluation of Large Language Models in Bangla Consumer Health Query Summarization](http://arxiv.org/abs/2505.05070v1), 2025-05-08

(39) [WaterDrum: Watermarking for Data-centric Unlearning Metric](http://arxiv.org/abs/2505.05064v1), 2025-05-08

(40) [CodeMixBench: Evaluating Large Language Models on Code Generation with Code-Mixed Prompts](http://arxiv.org/abs/2505.05063v1), 2025-05-08

(41) [Towards Mitigating API Hallucination in Code Generated by LLMs with Hierarchical Dependency Aware](http://arxiv.org/abs/2505.05057v1), 2025-05-08

(42) [LSRP: A Leader-Subordinate Retrieval Framework for Privacy-Preserving Cloud-Device Collaboration](http://arxiv.org/abs/2505.05031v1), 2025-05-08

(43) [A Reputation System for Large Language Model-based Multi-agent Systems to Avoid the Tragedy of the Commons](http://arxiv.org/abs/2505.05029v1), 2025-05-08

(44) [Scalable Multi-Stage Influence Function for Large Language Models via Eigenvalue-Corrected Kronecker-Factored Parameterization](http://arxiv.org/abs/2505.05017v1), 2025-05-08

(45) [The Pitfalls of Growing Group Complexity: LLMs and Social Choice-Based Aggregation for Group Recommendations](http://dx.doi.org/10.1145/3708319.3733659), 2025-05-08

(46) [Rethinking Invariance in In-context Learning](http://arxiv.org/abs/2505.04994v1), 2025-05-08

(47) [Latent Preference Coding: Aligning Large Language Models via Discrete Latent Codes](http://arxiv.org/abs/2505.04993v1), 2025-05-08

(48) [DenseGrounding: Improving Dense Language-Vision Semantics for Ego-Centric 3D Visual Grounding](http://arxiv.org/abs/2505.04965v1), 2025-05-08

(49) [Learning Item Representations Directly from Multimodal Features for Effective Recommendation](http://arxiv.org/abs/2505.04960v1), 2025-05-08

(50) [Chain-of-Thought Tokens are Computer Program Variables](http://arxiv.org/abs/2505.04955v1), 2025-05-08

(51) [Position: Epistemic Artificial Intelligence is Essential for Machine Learning Models to Know When They Do Not Know](http://arxiv.org/abs/2505.04950v1), 2025-05-08

(52) [Prompt-Based LLMs for Position Bias-Aware Reranking in Personalized Recommendations](http://arxiv.org/abs/2505.04948v1), 2025-05-08

(53) [Perception, Reason, Think, and Plan: A Survey on Large Multimodal Reasoning Models](http://arxiv.org/abs/2505.04921v1), 2025-05-08

(54) [An Open-Source Dual-Loss Embedding Model for Semantic Retrieval in Higher Education](http://arxiv.org/abs/2505.04916v1), 2025-05-08

(55) [SpatialPrompting: Keyframe-driven Zero-Shot Spatial Reasoning with Off-the-Shelf Multimodal Large Language Models](http://arxiv.org/abs/2505.04911v1), 2025-05-08

(56) [Theatrical Language Processing: Exploring AI-Augmented Improvisational Acting and Scriptwriting with LLMs](http://arxiv.org/abs/2505.04890v1), 2025-05-08

(57) [GroverGPT-2: Simulating Grover's Algorithm via Chain-of-Thought Reasoning and Quantum-Native Tokenization](http://arxiv.org/abs/2505.04880v1), 2025-05-08

(58) [From First Draft to Final Insight: A Multi-Agent Approach for Feedback Generation](http://arxiv.org/abs/2505.04869v1), 2025-05-08

(59) [PR2: Peephole Raw Pointer Rewriting with LLMs for Translating C to Safer Rust](http://arxiv.org/abs/2505.04852v1), 2025-05-07

(60) [Benchmarking LLM Faithfulness in RAG with Evolving Leaderboards](http://arxiv.org/abs/2505.04847v1), 2025-05-07

(61) [HiPerRAG: High-Performance Retrieval Augmented Generation for Scientific Insights](http://dx.doi.org/10.1145/3732775.3733586), 2025-05-07

(62) [Osiris: A Lightweight Open-Source Hallucination Detection System](http://arxiv.org/abs/2505.04844v1), 2025-05-07

(63) [Large Language Models are Autonomous Cyber Defenders](http://arxiv.org/abs/2505.04843v1), 2025-05-07

(64) [Putting the Value Back in RL: Better Test-Time Scaling by Unifying LLM Reasoners With Verifiers](http://arxiv.org/abs/2505.04842v1), 2025-05-07

(65) [Red Teaming the Mind of the Machine: A Systematic Evaluation of Prompt Injection and Jailbreak Vulnerabilities in LLMs](http://arxiv.org/abs/2505.04806v1), 2025-05-07

(66) [Safeguard-by-Development: A Privacy-Enhanced Development Paradigm for Multi-Agent Collaboration Systems](http://arxiv.org/abs/2505.04799v1), 2025-05-07

(67) [A Proposal for Evaluating the Operational Risk for ChatBots based on Large Language Models](http://arxiv.org/abs/2505.04784v1), 2025-05-07

(68) [When Bad Data Leads to Good Models](http://arxiv.org/abs/2505.04741v1), 2025-05-07

(69) [The Promise and Limits of LLMs in Constructing Proofs and Hints for Logic Problems in Intelligent Tutoring Systems](http://arxiv.org/abs/2505.04736v1), 2025-05-07

(70) [QBD-RankedDataGen: Generating Custom Ranked Datasets for Improving Query-By-Document Search Using LLM-Reranking with Reduced Human Effort](http://arxiv.org/abs/2505.04732v1), 2025-05-07

(71) [SOAEsV2-7B/72B: Full-Pipeline Optimization for State-Owned Enterprise LLMs via Continual Pre-Training, Domain-Progressive SFT and Distillation-Enhanced Speculative Decoding](http://arxiv.org/abs/2505.04723v1), 2025-05-07

(72) [Lay-Your-Scene: Natural Scene Layout Generation with Diffusion Transformers](http://arxiv.org/abs/2505.04718v1), 2025-05-07

(73) [Exploring Influence Factors on LLM Suitability for No-Code Development of End User IoT Applications](http://arxiv.org/abs/2505.04710v1), 2025-05-07

(74) [EchoInk-R1: Exploring Audio-Visual Reasoning in Multimodal LLMs via Reinforcement Learning](http://arxiv.org/abs/2505.04623v1), 2025-05-07

(75) [On Path to Multimodal Generalist: General-Level and General-Bench](http://arxiv.org/abs/2505.04620v1), 2025-05-07

(76) [OmniGIRL: A Multilingual and Multimodal Benchmark for GitHub Issue Resolution](http://arxiv.org/abs/2505.04606v1), 2025-05-07

(77) [MonoCoP: Chain-of-Prediction for Monocular 3D Object Detection](http://arxiv.org/abs/2505.04594v2), 2025-05-07

(78) [ZeroSearch: Incentivize the Search Capability of LLMs without Searching](http://arxiv.org/abs/2505.04588v1), 2025-05-07

(79) [SlideItRight: Using AI to Find Relevant Slides and Provide Feedback for Open-Ended Questions](http://arxiv.org/abs/2505.04584v1), 2025-05-07

(80) [Fight Fire with Fire: Defending Against Malicious RL Fine-Tuning via Reward Neutralization](http://arxiv.org/abs/2505.04578v1), 2025-05-07

(81) [Retrieval Augmented Generation Evaluation for Health Documents](http://arxiv.org/abs/2505.04680v1), 2025-05-07

(82) [Comparative Analysis of Carbon Footprint in Manual vs. LLM-Assisted Code Development](http://dx.doi.org/10.1145/3711919.3728678), 2025-05-07

(83) [Pangu Ultra MoE: How to Train Your Big MoE on Ascend NPUs](http://arxiv.org/abs/2505.04519v1), 2025-05-07

(84) [A Design Space for the Critical Validation of LLM-Generated Tabular Data](http://arxiv.org/abs/2505.04487v1), 2025-05-07

(85) [CAD-Llama: Leveraging Large Language Models for Computer-Aided Design Parametric 3D Model Generation](http://arxiv.org/abs/2505.04481v1), 2025-05-07

(86) [TrajEvo: Designing Trajectory Prediction Heuristics via LLM-driven Evolution](http://arxiv.org/abs/2505.04480v1), 2025-05-07

(87) [Miipher-2: A Universal Speech Restoration Model for Million-Hour Scale Data Restoration](http://arxiv.org/abs/2505.04457v1), 2025-05-07

(88) [M2Rec: Multi-scale Mamba for Efficient Sequential Recommendation](http://arxiv.org/abs/2505.04445v1), 2025-05-07

(89) [Towards Effectively Leveraging Execution Traces for Program Repair with Code LLMs](http://arxiv.org/abs/2505.04441v1), 2025-05-07

(90) [OBLIVIATE: Robust and Practical Machine Unlearning for Large Language Models](http://arxiv.org/abs/2505.04416v1), 2025-05-07

(91) [YABLoCo: Yet Another Benchmark for Long Context Code Generation](http://arxiv.org/abs/2505.04406v1), 2025-05-07

(92) [Large Means Left: Political Bias in Large Language Models Increases with Their Number of Parameters](http://arxiv.org/abs/2505.04393v1), 2025-05-07

(93) [The Aloe Family Recipe for Open and Specialized Healthcare LLMs](http://arxiv.org/abs/2505.04388v1), 2025-05-07

(94) [CDE-Mapper: Using Retrieval-Augmented Language Models for Linking Clinical Data Elements to Controlled Vocabularies](http://arxiv.org/abs/2505.04365v1), 2025-05-07

(95) [Benchmarking LLMs' Swarm intelligence](http://arxiv.org/abs/2505.04364v1), 2025-05-07

(96) [REVEAL: Multi-turn Evaluation of Image-Input Harms for Vision LLM](http://arxiv.org/abs/2505.04673v1), 2025-05-07

(97) [GASCADE: Grouped Summarization of Adverse Drug Event for Enhanced Cancer Pharmacovigilance](http://arxiv.org/abs/2505.04284v1), 2025-05-07

(98) [Weaponizing Language Models for Cybersecurity Offensive Operations: Automating Vulnerability Assessment Report Validation; A Review Paper](http://arxiv.org/abs/2505.04265v1), 2025-05-07

(99) [Steerable Chatbots: Personalizing LLMs with Preference-Based Activation Steering](http://arxiv.org/abs/2505.04260v1), 2025-05-07

(100) [CompileAgent: Automated Real-World Repo-Level Compilation with Tool-Integrated LLM-based Agent System](http://arxiv.org/abs/2505.04254v1), 2025-05-07
